lib/utils.js

```diff
@@ -108,7 +108,11 @@ const _keyColumns = (keys, alias) => {
     .map(key => ({ ref: [alias, key.name] }))
 }
 
-const _alias = entity => entity.name.replace(`${entity._service.name}.`, '').replace('.', '_')
+const _alias = entity => {
+  // REVISIT: we should not rely on entity._service (but I don't want to break existing behavior right now)
+  if (!entity._service) return entity.name.replace(/\./g, '_')
```

Comment:
Stringifying the `entity` should result in the same string, but as the other branch is still manipulating the name I would keep the code as is.

```suggestion
  if (!entity._service) return `${entity}`
```